src/lib.rs

```diff
@@ -107,6 +118,16 @@ impl Display for XorName {
     }
 }
 
+impl Distribution<XorName> for Standard {
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> XorName {
+        let mut ret = [0u8; XOR_NAME_LEN];
```

Comment:
I think we can just do `XorName(rng.gen())` here rather than iterating each `u8`.